src/ehttpc.appup.src

```diff
@@ -1,45 +1,77 @@
 %% -*-: erlang -*-
-{"0.4.0",
+{"0.4.1",
    [
+     {"0.4.0", [
+       {load_module, ehttpc, brutal_purge, soft_purge, []},
+       {load_module, ehttpc_sup, brutal_purge, soft_purge, []},
```

Comment:
Since the fixed child spec was changed, should we call `{update, ehttpc_sup, supervisor}` to update that?

Reply:
it will likely not work as the supervisor is not started by application controller.